WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -232,6 +232,7 @@ private extension ShippingLabelAddressFormViewController {
                 self.onCompletion(self.viewModel.address)
                 self.navigationController?.popViewController(animated: true)
             case .failure:
+                self.displayAddressUpdateFailedNotice()
                 break
```

Comment:
Nit: We can omit `break` if a `case` has content.

Reply:
Forgot to remove it, sorry 🤦 Thanks for the tip!